src/ions/utils/randomCardNumber.js

```diff
@@ -0,0 +1,4 @@
+const randomCardNumber = maxCard => {
+	return Math.floor(Math.random() * maxCard) + 1;
+};
+export default randomCardNumber;
```

Comment:
you probably could need some other util functions in the future, a named export could be better